### PR TITLE
[torch.nn Onboarding Lab] adding a new module 'Bias'

### PR DESCRIPTION
**Pull request by NivekT**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #62992

This PR is created as an onboarding exercise. This will not be merged with master.

### CODE REVIEW

**Review by jbschlosser — COMMENTED** [review]
Nice work!! just a couple small comments below but the implementation looks correct overall :)

File: test/test_nn.py [hunk 1]
```diff
@@ -17601,6 +17601,15 @@ def test_lazy_share_memory_buffer(self):
         with self.assertRaisesRegex(RuntimeError, 'share memory on an uninitialized'):
             module.share_memory()
 
+    def test_bias(self):
+        module = nn.Bias(5)
+        input = torch.ones(10, 5)
+        module(input)
+        self.assertIsInstance(module, nn.Bias)
```
[the remark this answers is not in the document]
Comment (NivekT) [reply test/test_nn.py:17608]: Removed.

File: test/test_nn.py [hunk 2]
```diff
@@ -17601,6 +17601,15 @@ def test_lazy_share_memory_buffer(self):
...
+        self.assertTrue(module.bias_values.shape == (5,))
+        y = module(input)
+        self.assertTrue(torch.equal(torch.nn.functional.bias(input, module.bias_values), y))
```
[the remark this answers is not in the document]
Comment (NivekT) [reply test/test_nn.py:17611]: Switched to `self.assertEqual()` and changing `y` to `y = input + module.bias_values`

File: torch/csrc/api/include/torch/nn/functional/linear.h [hunk 3]
```diff
@@ -26,6 +26,12 @@ inline Tensor linear(const Tensor& input, const Tensor& weight,
   }
 }
 
+// ============================================================================
+
+inline Tensor bias(const Tensor& input, const Tensor& bias_values) {
+  return input + bias_values;
```
[the remark this answers is not in the document]
Comment (NivekT) [reply torch/csrc/api/include/torch/nn/functional/linear.h:32]: Switched to calling core operator logic.

File: torch/nn/functional.py [hunk 4]
```diff
@@ -1827,6 +1827,17 @@ def hardsigmoid(input: Tensor, inplace: bool = False) -> Tensor:
         return torch._C._nn.hardsigmoid_(input)
     return torch._C._nn.hardsigmoid(input)
 
+def bias(input: Tensor, bias_values: Tensor) -> Tensor:
+    r""" Adds a tensor of normally distributed bias values to the last dimension
+    of the incoming data :math:`y = x + b`
+
+    Shape:
+        - Input: :math:`(*, H_{n})` where :math:`*` means any number of
```
[the remark this answers is not in the document]
Comment (NivekT) [reply torch/nn/functional.py:1835]: Yes, that shouldn't have any problem. I think the wording should be "including None"

File: torch/nn/functional.py [hunk 5]
```diff
@@ -1827,6 +1827,17 @@ def hardsigmoid(input: Tensor, inplace: bool = False) -> Tensor:
...
+          dimensions excluding none and :math:`H_{n} = \text{n\_features}`.
```
[the remark this answers is not in the document]
Comment (NivekT) [reply torch/nn/functional.py:1836]: Fixed.

File: torch/testing/_internal/common_nn.py [hunk 6]
```diff
@@ -98,6 +98,15 @@ def get_weight(m):
 # - `has_parity`: if `False`, expects this test dict to fail the C++ parity test. Default: True.
 
 module_tests = [
+    dict(
```
[the remark this answers is not in the document]
Comment (NivekT) [reply torch/testing/_internal/common_nn.py:101]: I see, thanks for pointing that out.

**Review by jbschlosser — COMMENTED** [review]
LGTM!

**Comment by NivekT** [comment]
This exercise PR looks good and can be closed.

**Pull request closed without merging**